src/components/pager/pager.component.js

```diff
@@ -162,53 +183,59 @@ const Pager = ({
         onLast={handleOnLast}
         onPagination={onPagination}
         pageCount={pageCount}
+        showFirstButton={showFirstButton}
+        showLastButton={showLastButton}
+        showNextButton={showNextButton}
+        showPreviousButton={showPreviousButton}
+        showPageCount={showPageCount}
       />
-      <StyledPagerSummary>
-        {totalRecords} {records(totalRecords)}
-      </StyledPagerSummary>
+      <StyledPagerSummary>{renderTotalRecords()}</StyledPagerSummary>
     </StyledPagerContainer>
   );
 };
 
 Pager.propTypes = {
   /** Function called when pager changes (PageSize, Current Page) */
   onPagination: PropTypes.func.isRequired,
-  /** Callback function for next link  */
+  /** Callback function for next link */
   onNext: PropTypes.func,
-  /** Callback function for first link  */
+  /** Callback function for first link */
   onFirst: PropTypes.func,
-  /** Callback function for previous link  */
+  /** Callback function for previous link */
   onPrevious: PropTypes.func,
-  /** Callback function for last link  */
+  /** Callback function for last link */
   onLast: PropTypes.func,
   /** Current visible page */
   currentPage: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
   /** Total number of records */
   totalRecords: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
   /** Pagination page size */
   pageSize: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
-  /** Should the page size selection dropdown be shown */
-  showPageSizeSelection: PropTypes.bool,
   /** Set of page size options */
   pageSizeSelectionOptions: PropTypes.arrayOf(
     PropTypes.shape({
       id: PropTypes.string,
       name: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
     })
   ),
-};
-
-Pager.defaultProps = {
-  currentPage: 1,
-  pageSize: 10,
-  totalRecords: 0,
-  showPageSizeSelection: false,
-  pageSizeSelectionOptions: [
-    { id: "10", name: 10 },
-    { id: "25", name: 25 },
-    { id: "50", name: 50 },
-    { id: "100", name: 100 },
-  ],
+  /** Should the page size selection dropdown be shown */
+  showPageSizeSelection: PropTypes.bool,
+  /** Should the label before the page size selection dropdown be shown */
+  showPageSizeLabelBefore: PropTypes.bool,
+  /** Should the label after the page size selection dropdown be shown */
+  showPageSizeLabelAfter: PropTypes.bool,
+  /** Should the total records label be shown */
+  showTotalRecords: PropTypes.bool,
+  /** Should the `First` navigation button be shown */
+  showFirstButton: PropTypes.bool,
```

Comment:
Same comment as above, I think `first` and `last` should be grouped, and `next` and `previous` should be grouped

Reply:
Yeah agree, the ticket said to make them controllable separately and the designs seemed to imply that but I'll see if UX are cool with grouping as it makes more sense

Reply:
Have spoken to UX will group the buttons as suggested